language/German/strings.po

```diff
@@ -512,15 +518,15 @@ msgstr "Automatisch speichern/wiederherstellen"
 
 #: src/frontend/mame/ui/submenu.cpp:77
 msgid "Allow rewind"
-msgstr ""
+msgstr "Zurückspulen zulassen"
 
 #: src/frontend/mame/ui/submenu.cpp:78
 msgid "Rewind capacity"
 msgstr "Rückspulfähigkeit"
 
 #: src/frontend/mame/ui/submenu.cpp:79
 msgid "Bilinear filtering for snapshots"
-msgstr ""
+msgstr "Bilineare Filterung für Schnappschüsse"
```

Comment:
"snapshots" is sometimes translated as "Bildschirmfotos" and sometimes as "Schnappschüsse". Why not stick to one?

Reply:
There is a single occurrance of _Bildschirmfotos_ I could find, which I changed to _Schnappschüsse_